src/AppBundle/RSS/DefaultTemplate.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+    namespace AppBundle\RSS;
+
+    /**
+      * DefaultTemplate class (inherit from Template)
+      * Common RSS template used by website
+      */
+    class DefaultTemplate extends Template {
+        static $ITEM_TAG = "item";
```

Comment:
Better use ``const`` than ``static``. A ``static`` property is public by default and then can be modified from outside. ``const`` cannot